lib/grell/rawpage.rb

```diff
@@ -36,6 +35,17 @@ def has_selector?(selector)
       page.has_selector?(selector)
     end
 
+    def finished_all_ajax_requests?(timeout, interval)
+      Timeout::timeout(timeout) do
```

Comment:
reader runs everything inside a timeout itself, so we have a timeout inside a timeout...